src/breadcrumb/breadcrumb.ts

```diff
@@ -42,12 +46,10 @@ export type BreadcrumbItem = {
             *ngIf="!item.isLast"
             (click)="handleClick($event, item)"
           >
-            {{item.name}}
+            {{ item.name }}
           </a>
 
-          <span *ngIf="item.isLast">
-            {{item.name}}
-          </span>
+          <span *ngIf="item.isLast"> {{ item.name }} </span>
```

Comment:
Any reasons why there are spaces within the `span`?

Reply:
that's the way prettier formatted it (the original was on two lines, which is equivalent to a space)